namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlServiceConfig.scala

```diff
@@ -28,11 +28,14 @@ case class HttpControlServable(
   addr: InetSocketAddress,
   store: DtabStore,
   delegate: Ns => NameInterpreter,
-  namers: Map[Path, Namer]
+  namers: Map[Path, Namer],
+  stats: StatsReceiver
 ) extends Servable {
   def kind = HttpControlServiceConfig.kind
   def serve(): ListeningServer = Http.server
+    .withLabel("httpController")
```

Comment:
what do you think about using the `kind` here?  (or is the kind not namespaced?) either way, maybe just reference the configId directly?